CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
 cmake_minimum_required(VERSION 3.13)
 project(coolstory_gram_client)
+include(FetchContent)
 
 set(CMAKE_CXX_STANDARD 17)
 
+find_package(Boost 1.74 REQUIRED)
+
+FetchContent_Declare(json URL https://github.com/nlohmann/json/releases/download/v3.11.2/json.tar.xz)
```

Comment:
I think we need to use more "safety" using of `FetchContent_Declare`:
```cmake
FetchContent_Declare(
        json
        GIT_REPOSITORY  https://github.com/nlohmann/json.git
        GIT_TAG         bc889afb4c5bf1c0d8ee29ef35eaaf4c8bef8a5d # release-3.11.2
)
```

Because release page or archive with json library can be removed, but commit not.

Reply:
I will fix it soon.